app/controllers/documentController.js

```diff
@@ -1,83 +1,28 @@
 'use strict';
 
-const models = require('../models/dbconnect');
+const models = require('../models/dbconnect'),
+  helpers = require('./helperMethods'),
+  helperMethods = new helpers(models);
 
-function handleError(res, reason, message, code) {
-  console.log('error:' + reason);
-  res.status(code || 500).json({ 'error': message });
-}
-
-function filterDocs(req, documents) {
-  if (!Array.isArray(documents)) {
-    documents = [documents];
-  }
-  return documents.filter((document) => {
-    const isPublic = document.public;
-    const isAdmin = req.decoded && req.decoded.id === 1;
-    const isOwner = req.decoded && req.decoded.id === document.ownerId;
-
-    return isPublic || isAdmin || isOwner;
-  });
-}
-
-function dateFilter(date, documents) {
-  return documents.filter((document) => {
-    return date === JSON.stringify(document.createdAt).substr(1, 10);
-  });
-}
-
-function roleFilter(role, documents) {
-  return documents.filter((document) => {
-    return role.toLowerCase() === document.Role.title.toLowerCase();
-  });
-}
-
-function filterSearch(query, documents) {
-  if (query.date) {
-    documents = dateFilter(query.date, documents);
-  }
-  if (query.role) {
-    documents = roleFilter(query.role, documents);
-  }
-  if (query.limit && documents.length > query.limit) {
-    documents.length = query.limit;
-  }
-
-  return documents;
-}
-
-function searchDocument(req, res) {
-  models.Document.findAll({
-    include: [{ model: models.Role },
-    { model: models.User, as: 'owner' }
-    ],
-    order: '"createdAt" DESC',
-    offset: req.query.start || 0
-  }).then((documents) => {
-    res.send(filterSearch(req.query, filterDocs(req, documents)));
-  }).catch((err) => {
-    handleError(res, err.message, 'Error fetching documents.');
-  });
-}
-
-class docControl {
-  getDocuments(req, res) {
+class DocControl {
+  getDocuments(req, res, next) {
```

Comment:
### Nitpick

You should leave a space before declaring the method. You did that in the `RolesController`, not sure why you din't make it consistent throughout.